wallet/reference/multichain-api.md

```diff
@@ -0,0 +1,396 @@
+---
+description: See the Multichain API reference.
+sidebar_custom_props:
+  flask_only: true
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Multichain API
+
+:::flaskOnly
+:::
+
+Dapps can call Multichain API [methods](#methods) to create and manage
+[CAIP-25](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md) connections with MetaMask.
+The API also provides [events](#events) that wallets can send to dapps to notify them of onchain or connection changes.
+
+:::note See also
+- [About the Multichain API](../concepts/multichain-api.md)
+- [Interact with multiple networks](../how-to/manage-networks/use-multichain.md)
+:::
+
+## Methods
+
+### `wallet_createSession`
+
+Creates a multichain connection with a wallet, authorizing that wallet with the specified set of scopes
+and properties.
```

Comment:
scopes, accounts, methods, notifications, and properties? not sure if too specific or not

Reply:
If accounts, methods, and notifications are *part of* a scope, then I think the shorter description suffices. The parameters section goes into detail about what's included.